anime/stores/userStore.ts

```diff
@@ -43,9 +45,29 @@ export const useUserStore = defineStore("user", {
 		// }
 	},
 	actions: {
-		storeAnimeId(id: any) {
+		storeAnimeId(id: number) {
 			this.animeId = id;
 		},
+		async getAllGenre() {
+			try {
+				const endpoint = "http://127.0.0.1:8000/genres";
+				const headers = {
+					"Content-Type": "application/json",
+				};
+
+				const options = {
+					method: "GET",
+					headers: headers,
+				};
+
+				const response = await fetch(endpoint, options);
+				const data = await response.json();
+
+				return data;
+			} catch (error) {
+				console.log(error);
+			}
+		},
```

Comment:
this is a example where you would type what the function is return it should return  a genre object. There should be a interface for genre.